src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -224,7 +224,7 @@ class BeneficialOwnersStep extends React.Component {
                             )}
                         </View>
                     )}
-                    <Text style={[styles.textStrong, styles.mb5]}>
+                    <Text style={[styles.textStrong, styles.mv5]}>
```

Comment:
Added a top margin to separate "Agreement" from the top checkboxes and make it look more like @shawnborton's [image](https://github.com/Expensify/App/issues/5343#issuecomment-923042108).